unicode-transforms.cabal

```diff
@@ -71,9 +71,9 @@ library
       Data.ByteString.UTF8.Normalize
       Data.Text.Normalize
       Data.Unicode.Types
+      Data.Unicode.Internal.Division
```

Comment:
I'm reluctant to expose this module. Could we possibly use an [internal library](https://www.haskell.org/cabal/users-guide/developing-packages.html#sublibs) for it?

Reply:
Earlier I was of the view to not expose `Internal` modules. But I have stopped caring about it now. It just became a hassle, and we also had some trouble with internal libraries. So in streamly now we have just exposed everything. If you care about it, you can give a try to the internal library.